Makefile

```diff
@@ -48,6 +48,12 @@ gen-sdk-dev: dev-env-check-binaries ## Generates a dev version of the Foundation
 		--config ./config/foundation_sdk.dev.yaml \
 		--parameters kind_registry_version=next,grafana_version=main
 
+.PHONY: gen-tests
+gen-tests: dev-env-check-binaries ## Generates the code described by tests schemas.
+	rm -rf generated
+	$(RUN_DEVBOX) go run ./cmd/cli/ generate \
+		--config ./config/foundation_sdk.tests.yaml
```

Comment:
Shouldn't have any reference to the equality tests? Looks like that it executes all tests but they are only for equality ones.

Reply:
This doesn't execute any test, it generates types that some tests depend on (just the equality tests for now, but we'll probably add more in the future)